### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Sarah Jallon- Jonas Troeltsch - Jeremiah Steiner - Simon Guggisberg

### DISCUSSION THREAD

**Discussion by sarsurgithub** [heig-vd-dai-course/heig-vd-dai-course#183] [Show and tell]
PW3

**Comment by BetterBlood** [comment]
> PW3

PW4 ?

**Reply by BetterBlood** [reply]
[vm atteignable avec ce lien](https://colorgrid.dedyn.io/)

**Reply by ludelafo** [reply]
Bonjour @BetterBlood, @sarsurgithub, @JonasTroeltsch et @GuggisbergSimon,

Sans vouloir mettre la pression à votre groupe, je vois avec `nslookup` que vous avez mis l'adresse IP privée de la machine virtuelle plutôt que son adresse publique, ce qui fait que nous ne pouvons pas accéder à votre application sans être connecté au VPN.

Normalement un simple changement d'enregistrement DNS devrait suffire !

**Reply by jonas-tr** [reply]
merci d'avoir vu ça!

**Reply by GuggisbergSimon** [reply]
cela devrait être résolu en faisant un `nslookup colorgrid.dedyn.io` maintenant, merci encore du heads up

**Comment by GuggisbergSimon** [comment]
pour le repo, le voici : https://github.com/JonasTroeltsch/DAI-PW4
le hash pour le rendu est : 9db5b3b1b224355ad6d6fe97c87959c2e29a9da3 (sujet à changement jusqu'à 13h)
